src/components/flash.scss

```diff
@@ -1,17 +1,17 @@
 .flash {
   position: fixed;
-  right: var(--rms-space-large);
+  right: var(--rm-space-large);
   display: flex;
   align-items: center;
-  top: var(--rms-space-large);
-  z-index: 10;
+  top: var(--rm-space-large);
+  z-index: var(--rms-z-index-alert-group);
   width: auto;
-  padding: var(--rms-space-x-small) var(--rms-space-medium);
-  font-size: var(--rms-font-small);
-  background: var(--rms-colors-white);
+  padding: var(--rm-space-x-small) var(--rm-space-medium);
+  font-size: var(--rm-font-small);
+  background: var(--rm-color-white);
   animation: animateMessage 5s normal forwards;
-  box-shadow: var(--rms-shadow-x-small);
-  border-radius: var(--rms-radius-small);
+  box-shadow: var(--rm-shadow-x-small);
+  border-radius: var(--rm-radius-small);
 
   border-left: solid 4px;
```

Comment:
@Jeremy-Walton Can we use our tokens here instead of the static styles for border size.